src/main.rs

```diff
@@ -1087,6 +1087,17 @@ fn main() {
                         binary with the Debug profile.")
                     .long("dbg-name")
                     .short("D"))
+                .arg(Arg::with_name("extension")
+                    .help("Builds the installer with a different extension")
```

Comment:
I try to avoid using "Build" or "Builds" for the installer since Cargo _builds_ the Rust binaries. Instead, I use "Create" or "Creates" for the installer. Please change the short help text to: `Creates an installer with a different file extension`. Please note the inclusion of the word, "file", to avoid confusion with WiX extensions